gateway/main.py

```diff
@@ -0,0 +1,12 @@
+from aiohttp import web
+
+from src.app import create_app
+
+
+def main():
+    app = create_app()
+    web.run_app(host='0.0.0.0', port=8000, app=app)
```

Comment:
host - localhost (127.0.0.1)
port - take from env variable
Make logs, that server is started and on which port
for example

```
started gateway service on {port} port
```